src/test/scala/io/qbeast/spark/utils/QbeastCompactionIntegrationTest.scala

```diff
@@ -111,112 +111,115 @@ class QbeastCompactionIntegrationTest extends QbeastIntegrationTestSpec {
   }
 
   it should "respect cube information" in withExtendedSparkAndTmpDir(
-    sparkConfWithSqlAndCatalog.set("spark.qbeast.compact.minFileSize", "1"))((spark, tmpDir) => {
+    sparkConfWithSqlAndCatalog.set("spark.qbeast.compact.minFileSizeInBytes", "1"))(
+    (spark, tmpDir) => {
 
-    val data = loadTestData(spark)
+      val data = loadTestData(spark)
```

Comment:
Careful with the identation here :(

Reply:
Code are contained within the brackets: `(spark, tmpDir) => {`